package.json

```diff
@@ -21,14 +21,17 @@
     "editor": "esy $EDITOR",
     "test": "echo 'No tests but that is ok for now...'"
   },
+  "resolutions": {
+    "**/@opam-alpha/ocamlbuild": "esy-ocaml/ocamlbuild#0.11.0-esy"
```

Comment:
This is needed b/c of [bug with `ocambuild -where`](https://github.com/ocaml/ocamlbuild/issues/69) which is fixed but not on opam yet. Anyway, it allows us to test selective dep resolution in yarn.